nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,92 @@
+const express = require("express");
```

Comment:
I would really recommend using a formatter to make it easier to read your code, like the built in one in [VSCode](https://code.visualstudio.com/docs/editor/codebasics#_formatting).